Source/CameraView/CameraView.swift

```diff
@@ -127,7 +127,14 @@ class CameraView: UIViewController {
       }, completion: { finished in
         self.captureSession.beginConfiguration()
         self.captureSession.removeInput(currentDeviceInput)
-        do { try self.captureSession.addInput(AVCaptureDeviceInput(device: self.captureDevice)) } catch {}
+
+        if self.captureDevice!.supportsAVCaptureSessionPreset(AVCaptureSessionPreset1280x720) {
+          self.captureSession.sessionPreset = AVCaptureSessionPreset1280x720
+        } else {
+          self.captureSession.sessionPreset = AVCaptureSessionPreset640x480
+        }
+
+        try! self.captureSession.addInput(AVCaptureDeviceInput(device: self.captureDevice))
```

Comment:
Why? for some reason It always leads to crash without even trying to go to "catch" part.

If user switches from front to back camera, a higher quality mode should be selected.